crates/router/src/db/events.rs

```diff
@@ -685,17 +791,30 @@ mod tests {
             .await
             .expect("Failed to create Mock store");
         let event_id = "test_event_id";
+        let (tx, _) = tokio::sync::oneshot::channel();
+        let app_state = Box::pin(routes::AppState::new(
```

Comment:
Shouldn't we be using `PostgresqlTest` here?

Reply:
AppState with storage is not necessary here.